### ISSUE TITLE
Importing Masks seems fragile

### ISSUE THREAD

**Issue by LupusUmbrae** [senbox-org/snap-desktop#21]
Hello,

I've just started playing with the masks manager and exported my mask information from one product and attempted to import it into another (same NetCDF specification), however because the exported mask information also includes the image size it silently fails if the sizes are different.

``` xml
  <Mask type="Maths">
    <NAME value="new_mask_1" />
    <MASK_RASTER_WIDTH value="512" />
    <MASK_RASTER_HEIGHT value="5916" />
    <DESCRIPTION value="confidence_in.coastline" />
    <COLOR red="255" green="0" blue="0" alpha="255" />
    <TRANSPARENCY value="0.5" />
    <EXPRESSION value="confidence_in.coastline" />
  </Mask>
```

The only way to get exported masks to import into another product seems to be to manually edit the xml files. Is there a way to make exporting/importing of masks not depends on the image size?

**Comment by marpet** [comment]
I came across this problem a while ago too. But since now no one else complaint this got a low priority.
Actually it is sufficient to remove the width and height tag from the mask file. The size is taken from the target scene when these tags are absent. So actually there is no reason to write the width and height. I've created an [issue](https://senbox.atlassian.net/browse/SNAP-598) for it. Hopefully we can address it soon.
